### PR TITLE
chore(deps): update dependencies system.io.abstractions to v19

### PR DESCRIPTION
**Pull request by Liam-Rougoor**
Merge of #2351 and #2352.

### CODE REVIEW

File: src/Directory.Packages.props [hunk 1]
```diff
@@ -25,7 +25,7 @@
     <PackageVersion Include="Microsoft.TestPlatform.ObjectModel" Version="17.2.0" />
     <PackageVersion Include="Microsoft.TestPlatform.Portable" Version="17.2.0" />
     <PackageVersion Include="Microsoft.TestPlatform.TranslationLayer" Version="17.2.0" />
-    <PackageVersion Include="System.IO.Abstractions" Version="17.2.26" />
+    <PackageVersion Include="System.IO.Abstractions" Version="19.1.5" />
```
Comment (rouke-broersma) [inline src/Directory.Packages.props:28]: Should switch to new package names

Comment (rouke-broersma) [reply src/Directory.Packages.props:28]: https://www.nuget.org/packages/TestableIO.System.IO.Abstractions

File: src/Directory.Packages.props [hunk 2]
```diff
@@ -25,7 +25,8 @@
     <PackageVersion Include="Microsoft.TestPlatform.ObjectModel" Version="17.2.0" />
     <PackageVersion Include="Microsoft.TestPlatform.Portable" Version="17.2.0" />
     <PackageVersion Include="Microsoft.TestPlatform.TranslationLayer" Version="17.2.0" />
-    <PackageVersion Include="System.IO.Abstractions" Version="17.2.26" />
+    <PackageVersion Include="TestableIO.System.IO.Abstractions" Version="19.1.5" />
+    <PackageVersion Include="TestableIO.System.IO.Abstractions.Wrappers" Version="19.1.5" />
```
Comment (rouke-broersma) [inline src/Directory.Packages.props:29]: Why do we suddenly need this package?

Comment (Liam-Rougoor) [reply src/Directory.Packages.props:29]: The one we used (System.IO.Abstractions) referenced both TestableIO's `Abstractions` and `Abstractions.Wrappers`. Without `Wrappers`, the build fails.

Comment (Liam-Rougoor) [reply src/Directory.Packages.props:29]: `Abstractions` contains the `IFileSystem` interface, `Abstractions.Wrappers` contains the `FileSystem` class.

Comment (Liam-Rougoor) [reply src/Directory.Packages.props:29]: `Abstractions.Wrappers` references `Abstractions`, so I can leave the latter out, good catch.

**Pull request merged**